src/services/bundle.service.js

```diff
@@ -153,24 +157,67 @@ async function uploadTransactionBundle(req, res) {
   return bundle;
 }
 
-async function uploadResourcesFromBundle(entries, headers, baseUrl, baseVersion, protocol, xprovenanceIncluded) {
+async function uploadResourcesFromBundle(entries, headers, xprovenanceIncluded, baseVersion) {
   const scrubbedEntries = replaceReferences(entries);
   // define headers to be included in axios call
   const entryHeaders = { 'Content-Type': 'application/json+fhir' };
   if (xprovenanceIncluded) {
     entryHeaders['X-Provenance'] = headers['x-provenance'];
   }
```

Comment:
The above lines can be deleted. They were only relevant to the axios request method.